I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/evts_generaux/rien.rpy", line 49, in script
    with Dissolve(.5)
  File "game/screens.rpy", line 1531, in execute
    screen valeurs_traits():
  File "game/screens.rpy", line 1531, in execute
    screen valeurs_traits():
  File "game/screens.rpy", line 1542, in execute
    $ adressePortrait = situation_.DeterminerPortrait()
  File "game/screens.rpy", line 1542, in <module>
    $ adressePortrait = situation_.DeterminerPortrait()
TypeError: DeterminerPortraitPersoPrincipal() takes exactly 2 arguments (3 given)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/evts_generaux/rien.rpy", line 49, in script
    with Dissolve(.5)
  File "renpy/ast.py", line 1397, in execute
    renpy.exports.with_statement(trans, paired)
  File "renpy/exports.py", line 1646, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "renpy/display/core.py", line 2610, in do_with
    clear=clear)
  File "renpy/display/core.py", line 3101, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs)
  File "renpy/display/core.py", line 3512, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "renpy/display/core.py", line 567, in visit_all
    d.visit_all(callback, seen)
  File "renpy/display/core.py", line 567, in visit_all
    d.visit_all(callback, seen)
  File "renpy/display/core.py", line 567, in visit_all
    d.visit_all(callback, seen)
  File "renpy/display/core.py", line 567, in visit_all
    d.visit_all(callback, seen)
  File "renpy/display/screen.py", line 432, in visit_all
    callback(self)
  File "renpy/display/core.py", line 3512, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "renpy/display/screen.py", line 443, in per_interact
    self.update()
  File "renpy/display/screen.py", line 631, in update
    self.screen.function(**self.scope)
  File "game/screens.rpy", line 1531, in execute
    screen valeurs_traits():
  File "game/screens.rpy", line 1531, in execute
    screen valeurs_traits():
  File "game/screens.rpy", line 1542, in execute
    $ adressePortrait = situation_.DeterminerPortrait()
  File "game/screens.rpy", line 1542, in <module>
    $ adressePortrait = situation_.DeterminerPortrait()
  File "despin/gen_vie/situation.py", line 41, in DeterminerPortrait
TypeError: DeterminerPortraitPersoPrincipal() takes exactly 2 arguments (3 given)

Windows-10-10.0.19041
Ren'Py 7.4.4.1439
destinRoiClovis 1.0
Wed Jun  9 23:37:01 2021
